frontend/src/containers/ConfigStep/DateRangePicker/validation.ts

```diff
@@ -0,0 +1,54 @@
+import { BasicConfigState } from '@src/context/config/configSlice';
+import dayjsSameOrBeforePlugin from 'dayjs/plugin/isSameOrBefore';
+import dayjsSameOrAfterPlugin from 'dayjs/plugin/isSameOrAfter';
+import dayjs, { Dayjs } from 'dayjs';
+
+dayjs.extend(dayjsSameOrBeforePlugin);
+dayjs.extend(dayjsSameOrAfterPlugin);
+
+// some SIGNIFICANT prerequisites :
+// dayjs(undefined).isValid() true
+// dayjs(null).isValid() false
+// dayjs('').isValid() false
+// if dayjsA or dayjsB either is invalid
+// all comparisons e.g. dayjsA.isBefore(dayjsB) always return false
```

Comment:
remove comments

Reply:
I think the comments is necessary to indicate the dayjs underlying wired comparison logic, explained why we add many `null` judgement, and transform empty input to `null`, not `undefiend` or `''`.

Reply:
Good point ,but this knowledge should belong to dayjs, not heartbeat